### PR TITLE
Cxi cube

### PR DESCRIPTION
**Pull request by ZryletTC**
Make things work for cubes in cxi. Tested in cxilv1118

### CODE REVIEW

File: producers/cube_config_cxi.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+# ##### FILTERS ##########################################################################
+# filters to apply to the data
+# format: list of [det (field), low, high, name]
+# 'filter1' is the standard name and will not be added to the h5 file name.
+filters = [
+    #['lightStatus/xray',0.5,1.5,'filter1'],
+    # ['ipm_dg1/sum',1e3,4.5e4,'filter1'],
+    ['ipm_dg2/sum',-1e6,4.5e4,'filter1'],
+    # ['ipm3/sum',200,3e3,'filter1'],
```
[the remark this answers is not in the document]
Comment (ZryletTC) [reply producers/cube_config_cxi.py:20]: why?

File: producers/cube_config_cxi.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+# ##### DETECTORS ########################################################################
+# List detectors to be cubed. Area detector have additional options such as threshold
+# Full images will always be saved. 
+# More area detector can be defined following the same syntax, and adding them to varList
+detDict = {'source':'jungfrau4M', 
+           'full':1, 
+           'image':1, 
+           'thresADU':-1e5, 
+           'common_mode':0}
```
[the remark this answers is not in the document]
Comment (silkenelson) [reply producers/cube_config_cxi.py:38]: 0 means pedestal subtraction, only for the Jungfrau is this equivalent to calib.

**Comment by silkenelson** [comment]
@ZryletTC Do you want to add the changes that we ended up using in the end (or wait until reprocessing is settled)?

**Comment by ZryletTC** [comment]
I'll add the current changes to keep it up to date but wait for the reprocessing discussion to finish up before merging.

**Comment by ZryletTC** [comment]
Closing in favor of Silke's work

**Pull request closed without merging**